services/frontend-service/pkg/cmd/server.go

```diff
@@ -18,6 +18,8 @@ package cmd
 
 import (
 	"context"
+	"net/http"
```

Comment:
why we care about import order ?

Reply:
Can we get a lint/autoformatter that fixes this automatically?

Reply:
In sre-standards we even have a linter for that.

There is no technical difference between the previous and the order in this PR, as far as I know. The difference is just to the reader: Golang standard library imports are in the top block, everything else is in the bottom block, everything is alphabetically ordered.

Reply:
There's one more thing to it:
if the order is fixed, then we can't have meaningless diffs, were the order is changed accidentally, or because of different preferences/settings. This keeps the history clean and meaningful.